ss/ss.go

```diff
@@ -24,23 +23,14 @@ const (
 	duration              = time.Second * 5
 )
 
-func CreateComDetailsFromNode(debugPod *debug.DebugPod, node *corev1.Node, tcpFile, udpFile *os.File) ([]types.ComDetails, error) {
-	ssOutTCP, err := debugPod.ExecWithRetry("ss -anpltH", interval, duration)
+func CreateComDetailsFromNode(debugPod *debug.DebugPod, node *corev1.Node) (res []types.ComDetails, ssOutTCP, ssOutUDP []byte, err error) {
```

Comment:
The func called CreateComDetailsFromNode but returning ss raw outputs?

Reply:
yes this is from lior i think also name is not good